Azure.ResourceManager.Core/ResourceGroupContainer.cs

```diff
@@ -32,17 +33,16 @@ internal ResourceGroupContainer(SubscriptionOperations subscription)
             ClientOptions.Convert<ResourcesManagementClientOptions>()).ResourceGroups;
 
         /// <summary>
-        /// Creates a new ResourceGroup.
+        /// Constructs an object used to create a resource group.
         /// </summary>
-        /// <param name="name"> The name of the ResourceGroup. </param>
-        /// <param name="location"> The location of the ResourceGroup. </param>
-        /// <returns> A response with the <see cref="ArmOperation{ResourceGroup}"/> operation for this resource. </returns>
-        public ArmOperation<ResourceGroup> Create(string name, Location location)
+        /// <param name="location"> The location of the resource group. </param>
+        /// <returns> A builder with <see cref="ResourceGroup"> and <see cref="ResourceGroupData"/>. </returns>
+        public ArmBuilder<ResourceGroup, ResourceGroupData> Construct(Location location, IDictionary<string, string> tags = default, string managedBy = default)
         {
-            var model = new ResourceGroupData(new ResourceManager.Resources.Models.ResourceGroup(location));
-            return new PhArmOperation<ResourceGroup, Azure.ResourceManager.Resources.Models.ResourceGroup>(
-                Operations.CreateOrUpdate(name, model),
-                g => new ResourceGroup(Parent, new ResourceGroupData(g)));
+            var model = new ResourceManager.Resources.Models.ResourceGroup(location);
+            model.Tags = tags;
```

Comment:
thought tags was unassignable, how does this work?

Reply:
Apparently for resource group it is?

Is that a mistake?

Reply:
could be old library reference

Reply:
Should we changes this then?